.github/workflows/test_frontend_unit_and_qc.yml

```diff
@@ -0,0 +1,37 @@
+name: Unit tests and QC - Frontend
```

Comment:
## New unit test action
- [ ] Check that the action runs (`workflow_dispatch`)
- [ ] Works on pull request?
- [ ] Works on push to develop?
  - What does that look like? Is there a badge that will appear on the homepage, or the action will just run?

How I test this: (i) Create action w/ the same name on `develop` (empty or copy paste). Then I should be able to run action on this branch.

Reply:
Jest tests are passing in the action! https://github.com/jhu-bids/TermHub/actions/runs/7908022233/job/21586342511

I do see it running when I push to develop. itll be cool to see this on PR

Reply:
### Playwright
At first I was annoyed because Playwright tests were failing for same reason Jest ones were failing before, because of `  message: "SyntaxError: Unexpected token 'export'",`: https://github.com/jhu-bids/TermHub/actions/runs/7908023169/job/21586345829

I did little other than upgrade Playwright from 1.39 to 1.41, and now the issue is resolved: https://github.com/jhu-bids/TermHub/actions/runs/7908441846/job/21587623534